utils/gpsd/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cgps -V 2>&1 | grep "$2"
```

Comment:
```suggestion
gpsd -V 2>&1 | grep "$2"
```

Reply:
gpsd does not have a version option.

Reply:
```shell
root@dba40b1634ab:/# gpsd -V
gpsd: 3.20 (revision 3.20)
```

Reply:
So this package results in both a `gpsd` and a `gpsd-clients` package. To test it a script like the following should work:

```shell
case "$1" in
	"gpsd")
		gpsd -V 2>&1 | grep "$2"
		;;
        "gpsd-clients")
                cgps -V 2>&1 | grep "$2"
                ;;
esac
```

Reply:
Hmm maybe it was changed. gpsd -V locally returns nothing (version 3.19).